src/utils/store.ts

```diff
@@ -94,3 +95,8 @@ export const [restoreResetDevice, storeResetDevice] = fromStore(
     'resetDevice',
     false
 );
+
+export const [restoreRefreshOnStart, storeRefreshOnStart] = fromStore(
+    'refreshOnStart',
+    false
```

Comment:
In order to support the Quick Start project, we want it to refresh on start by default.

```suggestion
    true
```